blazor-wasm/obj/WebApplication1.csproj.nuget.cache

```diff
@@ -0,0 +1,5 @@
+{
+  "version": 1,
+  "dgSpecHash": "NDuF6YjtcCoYqH0XbROA+FgH7zDmqssPRAdSQCy8cZ7aXxpdQUXARuoTOz6FDYThRBRWOs3lpPIjDvsFSuQF3A==",
+  "success": true
+}
```

Comment:
**/obj/** is a dynamically generated folder and should not be committed to a source control